angular/main-es5.js

```diff
@@ -0,0 +1 @@
on",1),$.zc(5,"mat-checkbox"),$.dd(6,"Check me!"),$.yc(),$.yc(),$.yc(),$.yc())},directives:[C.a,C.b,S.a],styles:[""]}),a),re=((o=function e(){_classCallCheck(this,e),this.checked=!1,this.indeterminate=!1,this.labelPosition="after",this.disabled=!1}).\u0275fac=function(e){return new(e||o)},o.\u0275cmp=$.nc({type:o,selectors:[["checkbox-configurable-example"]],decls:27,vars:8,consts:[[1,"example-h2"],[1,"example-section"],[1,"example-margin",3,"ngModel","ngModelChange"],[1,"example-margin"],[3,"ngModel","ngModelChange"],["value","after",1,"example-margin"],["value","before",1,"example-margin"],[1,"result"],[1,"example-margin",3,"ngModel","indeterminate","labelPosition","disabled","ngModelChange","indeterminateChange"]],template:function(e,t){1&e&&($.zc(0,"mat-card"),$.zc(1,"mat-card-content"),$.zc(2,"h2",0),$.dd(3,"Checkbox configuration"),$.yc(),$.zc(4,"section",1),$.zc(5,"mat-checkbox",2),$.Hc("ngModelChange",(function(e){return t.checked=e})),$.dd(6,"Checked"),$.yc(),$.zc(7,"mat-checkbox",2),$.Hc("ngModelChange",(function(e){return t.indeterminate=e})),$.dd(8,"Indeterminate"),$.yc(),$.yc(),$.zc(9,"section",1),$.zc(10,"label",3),$.dd(11,"Align:"),$.yc(),$.zc(12,"mat-radio-group",4),$.Hc("ngModelChange",(function(e){return t.labelPosition=e})),$.zc(13,"mat-radio-button",5),$.dd(14,"After"),$.yc(),$.zc(15,"mat-radio-button",6),$.dd(16,"Before"),$.yc(),$.yc(),$.yc(),$.zc(17,"section",1),$.zc(18,"mat-checkbox",2),$.Hc("ngModelChange",(function(e){return t.disabled=e})),$.dd(19,"Disabled"),$.yc(),$.yc(),$.yc(),$.yc(),$.zc(20,"mat-card",7),$.zc(21,"mat-card-content"),$.zc(22,"h2",0),$.dd(23,"Result"),$.yc(),$.zc(24,"section",1),$.zc(25,"mat-checkbox",8),$.Hc("ngModelChange",(function(e){return t.checked=e}))("indeterminateChange",(function(e){return t.indeterminate=e})),$.dd(26," I'm a checkbox "),$.yc(),$.yc(),$.yc(),$.yc()),2&e&&($.hc(5),$.Qc("ngModel",t.checked),$.hc(2),$.Qc("ngModel",t.indeterminate),$.hc(5),$.Qc("ngModel",t.labelPosition),$.hc(6),$.Qc("ngModel",t.disabled),$.hc(7),$.Qc("ngModel",t.checked)("indeterminate",t.indeterminate)("labelPosition",t.labelPosition)("disabled",t.disabled))},directives:[C.a,C.b,S.a,c.g,c.i,F.b,F.a],styles:[".example-h2[_ngcontent-%COMP%]{margin:10px}.example-section[_ngcontent-%COMP%]{display:flex;align-content:center;align-items:center;height:60px}.example-margin[_ngcontent-%COMP%]{margin:0 10px}"]}),o);n.d(t,"AppModule",(function(){return ae}));var oe,ae=((oe=function e(){_classCallCheck(this,e)}).\u0275mod=$.rc({type:oe,bootstrap:[ne,re]}),oe.\u0275inj=$.qc({factory:function(e){return new(e||oe)},providers:[{provide:te.a,useValue:{appearance:"fill"}}],imports:[[l.a,f.b,c.c,i.b,ee,s.l,c.j]]}),oe);Object(u.a)().bootstrapModule(ae).catch((function(e){return console.error(e)}))},zn8P:function(e,t){function n(e){return Promise.resolve().then((function(){var t=new Error("Cannot find module '"+e+"'");throw t.code="MODULE_NOT_FOUND",t}))}n.keys=function(){return[]},n.resolve=n,e.exports=n,n.id="zn8P"}},[[0,0,5]]]);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['T' used outside of binding context.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5245121)